lib/core/util/text_constants.dart

```diff
@@ -13,4 +14,11 @@ abstract class TextConstants {
   static const String socialMediaTitle = 'Follow us';
   static const String apiExceptionMessage = 'response status not 200';
   static const String notFoundMessage = 'No recipes found';
+  static const String homeRoute = '/';
+  static const String detailsRoute = '/details';
+  static const String searchRoute = '/search';
+  static const String splashScreenRoute = '/splashScreen';
```

Comment:
As a suggestion, we usually use the path '/' as a reference to the splash

Reply:
Changed '/' to '/home' and '/splashScreen' to '/'